src/Symfony/Component/Security/Http/Tests/LoginLink/LoginLinkHandlerTest.php

```diff
@@ -48,41 +48,51 @@ protected function setUp(): void
      * @dataProvider provideCreateLoginLinkData
      * @group time-sensitive
      */
-    public function testCreateLoginLink($user, array $extraProperties)
+    public function testCreateLoginLink($user, $locale, array $extraProperties)
```

Comment:
But locale at last and add default null, so only one new test needs to be added to the provider, cause null is the default

Reply:
Ok. I will correct this tomorrow morning. Thanks for approval 🙂